### PR TITLE
Add Kleinanzeigen OAuth Domains to Docs

### PR DESCRIPTION
**Pull request by shanecolfer**

### CODE REVIEW

File: src/source/reference/get-regions.rst [hunk 1]
```diff
@@ -36,7 +36,6 @@ Errors
 ====================    ====    =======================     ==============================================================================
 Field                   Code    Error message               Description
 ====================    ====    =======================     ==============================================================================
-regionIds               2000    missing argument            The parameter needs to be present
```
Comment (twessling-icas) [inline src/source/reference/get-regions.rst]: Why is this removed? on the GET /regions I think the regionIds is quite mandatory

**Review by twessling-icas — COMMENTED** [review]
Looks good, but it looks like you didn't re-generate. Also the regions change seems fishy

**Review by twessling-icas — CHANGES_REQUESTED** [review]
I think we're at 0.126 already, please fix version and remove 0.123 again :D

**Comment by shanecolfer** [comment]
> 

I know I've fecked up this branch somehow :o, fixing now!

**Review by twessling-icas — APPROVED** [review]
